doc/programming_guide/migration.rst

```diff
@@ -2,13 +2,23 @@
 
 Migrating from pyglet 2.0
 =========================
-pyglet 2.1 contains several small, but breaking changes. Some of these are
-usibility improvements, and some are in the interest of improving the quality
-of the code base. If you are upgrading from pyglet 2.0 and your game or project
-has suddenly stopped working, this is the place for you. The following sections
-should hopefully get you up and running again without too much effort. If you
-are having an issue that is not covered here, please open up an issue ticket on
-`GitHub <https://github.com/pyglet/pyglet/issues>`_ so that we can add it.
+This page helps projects upgrade from pyglet 2.0.
```

Comment:
Just a suggestion on these few sentences: 

> This page is provided to help projects upgrade from pyglet 2.0. The new 2.1 series has several small, but breaking changes. These include improvements to:

Reply:
I've started redoing this section locally. I'll try to get commits pushed tonight or tomorrow, but changes are they won't be cleanly separated.